### PR TITLE
[CAM-8425] Review CAM Management > Providers

### PR DESCRIPTION
**Pull request by dani-cl-madrid**
Review CAM Management > Providers KB documentation and update it accordingly
Update providers table.
Updated metadata.
Included intro sections Table of Contents, overview, audience prerequisites (TCOAP).
Minor markdown fixes.

Please review @gsanchezu

### CODE REVIEW

File: Cloud Application Manager/Core Concepts/providers.md [hunk 1]
```diff
@@ -1,29 +1,54 @@
 {{{
 "title": "Providers",
-"date": "06-16-2017",
+"date": "10-17-2018",
 "author": "",
+"keywords": "CAM, ALM, Providers, CenturyLink Cloud, AWS, Azure, Google Cloud, CloudStack, Dimension Data, OpenStack, Rackspace, SoftLayer, VCloud, VMWare",
```
Comment (gsanchezu) [inline Cloud Application Manager/Core Concepts/providers.md]:
```suggestion
"keywords": ["cam", "alm", "providers", "centuryLink-cloud", "clc", "aws", "azure", "google-cloud", "gcp", "cloudstack", "dimension-data", "openstack", "rackspace", "softlayer", "vcloud", "vmware"],
```

File: Cloud Application Manager/Core Concepts/providers.md [hunk 2]
```diff
@@ -1,29 +1,54 @@
...
+
+All Cloud Application Manager customers.
+
+### Prerequisites
+
+An active Cloud Application Manager account.
+
 ### Providers
 
 A provider is a public or private cloud account you register in Cloud Application Manager. Cloud Application Manager interfaces with the provider account’s API to provision and orchestrate deployments automatically. Before you can deploy workloads from Cloud Application Manager, you need to register a provider first. Cloud Application Manager integrates with many popular clouds. Here’s a full list of clouds Cloud Application Manager supports and their availability by Edition:
```
Comment (gsanchezu) [inline Cloud Application Manager/Core Concepts/providers.md]: I suggest to add a new section called "Supported provider types" and include there the last sentence of the previous paragraph and the table below. Please also add it to TOC.

File: Cloud Application Manager/Core Concepts/providers.md [hunk 3]
```diff
@@ -1,29 +1,54 @@
...
-| [OpenStack Cloud](../Deploying Anywhere/using-openstack-cloud.md) | ✓ | ✓ |
-| [CloudStack](../Deploying Anywhere/using-cloudstack.md) | ✓ | ✓ |
-| [Rackspace Cloud](../Deploying Anywhere/using-rackspace-cloud.md) | ✓ | ✓ |
-| [SoftLayer](../Deploying Anywhere/using-softlayer.md) | ✓ | ✓ |
-
+| Clouds                                                                                                                | Enterprise <br> (Saas) | Enterprise <br> (Appliance) |
+| --------------------------------------------------------------------------------------------------------------------- | :--------------------: | :-------------------------: |
+| [CenturyLink Cloud](../Deploying%20Anywhere/using-centurylink-cloud.md)                                               | ✓                      | ✓                           |
+| [CenturyLink DCC](../Deploying%20Anywhere/using-dcc.md)                                                               | ✓                      | ✓                           |
+| [CenturyLink Private Cloud on VMware Cloud Foundation](../../centurylink-private-cloud-on-vmware-cloud-foundation.md) | ✓                      |                             |
```
Comment (gsanchezu) [inline Cloud Application Manager/Core Concepts/providers.md]: Any reason why you removed the tick in Appliance?

File: Cloud Application Manager/Core Concepts/providers.md [hunk 4]
```diff
@@ -1,29 +1,54 @@
...
+| [Amazon Web Services](../Deploying%20Anywhere/using-your-aws-account.md)                                              | ✓                      | ✓                           |
+| Amazon Web Services GovCloud (Same documentation as [Amazon Web Services](../Deploying%20Anywhere/using-your-aws-account.md)) |                | ✓                           |
```
Comment (gsanchezu) [inline Cloud Application Manager/Core Concepts/providers.md]: Any reason why you removed the tick in SaaS?

File: Cloud Application Manager/Core Concepts/providers.md (hunk 3 above)
Comment (dani-cl-madrid) [reply Cloud Application Manager/Core Concepts/providers.md]: When I added that service I checked in my local (applicance version) of CAM and couldn't find the provider.

File: Cloud Application Manager/Core Concepts/providers.md (hunk 4 above)
Comment (dani-cl-madrid) [reply Cloud Application Manager/Core Concepts/providers.md]: When I added that service I checked in the Mission Field corp (SAAS version) of CAM I couldn't find the provider. Neighter in the settings page.

File: Cloud Application Manager/Core Concepts/providers.md (hunk 2 above)
Comment (dani-cl-madrid) [reply Cloud Application Manager/Core Concepts/providers.md]: Done in a previous update.

File: Cloud Application Manager/Core Concepts/providers.md [hunk 5]
```diff
@@ -1,29 +1,56 @@
...
-| [Microsoft Azure](../Deploying Anywhere/using-azure.md) | ✓ | ✓ |
-| [VMware vCenter](../Deploying Anywhere/using-the-vmware-vcenter-private-datacenter.md) | ✓ | ✓ |
-| [vCloud Air and vCloud Director](../Deploying Anywhere/orchestrating-vcloud-air-vcloud-director-deployments.md) | ✓ | ✓ |
-| [OpenStack Cloud](../Deploying Anywhere/using-openstack-cloud.md) | ✓ | ✓ |
-| [CloudStack](../Deploying Anywhere/using-cloudstack.md) | ✓ | ✓ |
-| [Rackspace Cloud](../Deploying Anywhere/using-rackspace-cloud.md) | ✓ | ✓ |
-| [SoftLayer](../Deploying Anywhere/using-softlayer.md) | ✓ | ✓ |
+#### Supported provider types
 
+Here’s a full list of clouds Cloud Application Manager supports and their availability by Edition:
```
[the remark this answers is not in the document]
Comment (dani-cl-madrid) [reply Cloud Application Manager/Core Concepts/providers.md]: Don't worry. Done.

**Pull request merged**